wolfssl/wolfcrypt/tfm.h

```diff
@@ -70,11 +70,6 @@
       #define TFM_X86_64
    #endif
 #endif
-#if defined(__aarch64__)
```

Comment:
Why did this need to come out?
Is it causing failures on ARM builds?

Reply:
It causes failure with the linaro board. Using FP_64BIT alone with previous releases also fail on the linaro board. The failure happens with RSA and testwolfcrypt

Reply:
OK.